doc/rst/source/cookbook/map-projections.rst

```diff
@@ -404,7 +404,7 @@ Perspective projection (**-Jg** **-JG**)
 
 **Syntax**
 
-    **-Jg**\|\ **G**\ *lon0/lat0*\ */*\ *scale*\|\ *width*\ [**+a**\ *azimuth*][**+t**\ *tilt*][**+v**\ *vwidth/vheight*][**+w**\ *twist*][**+z**\ *altitude*]
+    **-Jg**\|\ **G**\ *lon0/lat0*\ */*\ *scale*\|\ *width*\ [**+a**\ *azimuth*][**+t**\ *tilt*][**+v**\ *vwidth/vheight*][**+w**\ *twist*][**+z**\ *altitude*\ [**r**\|\ **R**]\|\ **g**]
```

Comment:
```suggestion
    **-Jg**\|\ **G**\ *lon0/lat0*\ */*\ *scale*\|\ *width*\ [**+a**\ *azimuth*][**+t**\ *tilt*][**+v**\ *vwidth/vheight*][**+w**\ *twist*][**+z**\ *altitude*\ [**r**\|\ **R**\|\ **g**]]
```
Is *altitude* **Rg** allowed? My interpretation is that it should be **r** or **R** or **g**.

Reply:
No, the four ways are _altitude_, _altitude_**r**, _altitude_**R**, and **g**.  I thought that is how one would interepret that syntax, no?

Reply:
OK, I misread.